src/Dotnet.Script.DependencyModel/ProjectSystem/FileUtils.cs

```diff
@@ -41,8 +43,25 @@ public static string GetPathToTempFolder(string targetDirectory)
 
                 targetDirectoryWithoutRoot = Path.Combine(driveLetter, targetDirectoryWithoutRoot);
             }
-            var pathToProjectDirectory = Path.Combine(tempDirectory, "scripts", targetDirectoryWithoutRoot);
+            var pathToProjectDirectory = Path.Combine(tempDirectory, "dotnet-script", targetDirectoryWithoutRoot);
             return pathToProjectDirectory;
         }
+
+        public static string GetTempPath()
+        {
+            var userFolder = System.Environment.GetFolderPath(System.Environment.SpecialFolder.UserProfile);
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
Shouldn't we be using `$XDG_CACHE_HOME`, if set?

Reply:
yes, you are right - it will be handled in another PR 👍